src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -535,13 +535,13 @@ well as menu structures (for main menu and popup menus).
          <shortcut value="Alt+Up" title="Move Lines Up" />
          <shortcut value="Alt+Down" title="Move Lines Down" />
          <shortcut value="Ctrl+D" title="Delete Line" if="!org.rstudio.core.client.BrowseCap.isMacintosh()" />
```

Comment:
Does this mean we've gotten rid of the Delete Line shortcut entirely on the Mac?